rMATS_pipeline/rmatspipeline/rmatspipeline.pyx

```diff
@@ -303,21 +306,21 @@ cdef int is_bam_multijunc(const BamAlignment& bread,
     """
     cdef:
         size_t i
-        size_t num = bread.CigarData.size()
+        size_t num = cigar_data.size()
         int length = 0
 
     if num < 3 or num % 2 == 0:
         return READ_NOT_EXPECTED_CIGAR
 
     for i in range(num):
         if i % 2 == 1:
-            if bread.CigarData[i].Type != 'N':
+            if cigar_data[i].Type != 'N':
                 return READ_NOT_EXPECTED_CIGAR
         else:
-            if bread.CigarData[i].Type != 'M':
+            if cigar_data[i].Type != 'M':
                 return READ_NOT_EXPECTED_CIGAR
 
-            length += bread.CigarData[i].Length
+            length += cigar_data[i].Length
 
     if (not variable_read_length) and length != readLength:
```

Comment:
Same here, the comparison with input readLength should be done before dropping the soft/hard clipping BamAlignment data.

Reply:
The latest commit includes the clipped portion when comparing with readLength